css/main.css

```diff
@@ -1,3 +1,667 @@
+  font-size: 23px;
+}
+
+/* Introduction */
+.introduction {
+  text-align: center;
+  background-color: var(--beige-light);
+  position: absolute;
+  width: 90%;
+  top: 120px;
+  left: 27px;
+}
+.introduction .row {
+  padding: 1rem 0rem;
+}
+.introduction p {
+  display: none;
+}
+.introduction img {
+  margin-top: 1.5rem;
+  border-radius: 50%;
+}
+.introduction h2 {
+  margin-top: 1rem;
+  font-family: var(--heading-font);
+}
+.introduction .blue-line {
+  width: 80px;
+  border-bottom: 2px solid var(--blue-royal);
+  margin: 0 auto;
+  margin-top: 0.5rem;
+}
+.introduction .btns {
+  margin-top: 2rem;
+}
+.introduction .btn-secondary {
+  margin-left: 1rem;
+}
+.introduction .col-icons {
+  margin-top: 2.5rem;
+  padding: 1rem 0rem;
+  background: var(--white);
+}
+.introduction .col-icons .social-icons {
+  display: flex;
+  justify-content: center;
+}
+.introduction .col-icons .social-icons li {
+  padding: 0rem 1rem;
+}
+/* ---------------------------- */
+
+/* Main - Resume Page */
+.second-nav {
+  height: 15vh;
+}
+.resume {
+  background: var(--beige-dark);
+  padding-left: 2rem;
+  padding-bottom: 2rem;
+  line-height: 1.7;
+}
+.resume h1 {
+  font-size: 40px;
+}
+.resume .title {
+  padding-top: 4rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.resume .square,
+.projects .square,
+.contact .square {
+  width: 25px;
+  height: 25px;
+  margin-right: 0.75rem;
+}
+.resume .exp-title {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 2rem;
+  font-size: 18px;
+}
+.resume .exp-title .btn {
+  padding: 0.5rem 1rem;
+  margin-right: 0.5rem;
+}
+.resume article,
+.resume .skill {
+  background-color: var(--white);
+  width: 97%;
+  padding: 2rem 1rem 2rem 2rem;
+  margin: 3.5rem 0rem;
+}
+.resume .title-art1,
+.resume .title-art2 {
+  margin-bottom: 1.5rem;
+}
+.resume .skill h3 {
+  margin-bottom: 1rem;
+}
+.resume ul .square {
+  width: 17px;
+  height: 17px;
+}
+.resume ul li {
+  display: flex;
+  align-items: center;
+}
+
+/* Main - Project Page */
+.projects {
+  background-color: var(--beige-dark);
+  padding: 4rem 0rem 2rem 2rem;
+  line-height: 1.7;
+}
+.projects .title {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.projects .title ~ p {
+  margin-top: 1.5rem;
+  margin-right: 0.5rem;
+}
+
+.projects .pro-articles {
+  width: 97%;
+  margin-top: 3rem;
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+}
+.projects .pro-articles .article {
+  background-color: var(--white);
+  /* padding-top: 1.3rem; */
+  display: flex;
+  flex-direction: column;
+}
+.projects .pro-articles .article .col {
+  flex: 1;
+}
+.projects .pro-articles .article .col1 {
+  padding-top: 1.3rem;
+}
+
+.projects .article .article-title {
+  border-left: 12px solid var(--blue-royal);
+  padding-left: 1.2rem;
+}
+.projects .article .article-title ~ p {
+  padding: 2rem 1.2rem 1.5rem 1.2rem;
+}
+.projects .article img {
+  width: 100%;
+}
```

Comment:
let's add `height: 100%` to remove the white space at the bottom of the images

<img width="500" alt="스크린샷 2023-03-06 00 08 33" src="https://user-images.githubusercontent.com/63498427/222994056-4acf0a5d-3911-4bc6-98c0-dc19c81ea7e5.png">